Track Buddy/MotionManager.swift

```diff
@@ -12,32 +12,57 @@ import Collections
 import CoreGraphics
 
 class MotionManager: ObservableObject {
+    private enum Parameters {
+        static let deviceMotionUpdateInterval: TimeInterval = 1/100
+        static let graphUpdateInterval = RunLoop.SchedulerTimeType.Stride(1/15)
+        static var pointStorageLimit: Int {
+            Int(1 / graphUpdateInterval.magnitude * 3)
+        } // final number after multiplier == number of seconds retained for tracer line
```

Comment:
nit- comment should go above line 19

Reply:
I get what you're going for, but it's not very common. 

Could be misconstrued as "should there be another / missing line after this comment?"